PyInstaller/utils/hooks/qt.py

```diff
@@ -48,11 +48,11 @@ def __getattr__(self, name):
                 sys.stderr = sys.stdout
 
                 import json
-                from %s.QtCore import QLibraryInfo, QCoreApplication
```

Comment:
Please remove this change. It doesn't follow PyInstaller coding conventions.

Reply:
I agree.